manifests/m/MilosParipovic/OneCommander/3.44.1/MilosParipovic.OneCommander.locale.en-US.yaml

```
@@ -0,0 +1,24 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: MilosParipovic.OneCommander
+PackageVersion: 3.44.1
+PackageLocale: en-US
+Publisher: Milos Paripovic
+PublisherUrl: https://onecommander.com/
+PublisherSupportUrl: https://twitter.com/OneCommander

```

```suggestion
PublisherSupportUrl: https://onecommander.com/help
```
A social media account is not considered as the